### PR TITLE
Cross(X) shape added

### LINKED ISSUE #52: Cross Shape
I can add a cross (X) shape.
Please Assign it to me!

### PR DESCRIPTION
**Pull request by Spyware007**
Closes #52 
Feature added #52 
Please review and merge

### CODE REVIEW

File: src/lib/shapes/cross.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  top = ``,
+  position = `absolute`,
+  left = ``,
+  right = ``,
+  bottom = ``,
+  className = `anim-shape-star`,
+  zIndex = `-1`,
+  breakpoints = [600, 900, 1200]
+}: CrossProps) => {
+  const StyledStar = styled('div')`
```
Comment (Sachin-chaurasiya) [inline src/lib/shapes/cross.tsx:31]:
```suggestion
  const StyledCross = styled('div')`
```

File: src/lib/shapes/data/crossPath.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export const crossPath: any = `polygon(
```
Comment (Sachin-chaurasiya) [inline src/lib/shapes/data/crossPath.ts:1]:
```suggestion
export const crossPath: string = `polygon(
```

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]
Few changes are required.

**Comment by Spyware007** [comment]
Done with the changes @Sachin-chaurasiya  Please review and merge

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**